lib/challenge.js

```diff
@@ -48,7 +57,13 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name:'Baxter',
+  species: 'dog',
+  breed:'Golden Retriver'
```

Comment:
There is no comma on this line, which is required to separate key-value pairs. This object literal, when parsed, will cause the interpreter to throw an error between `'Golden Retriver'` and `noise:`.